code/app/src/main/res/layout/entrant_home_view.xml

```diff
@@ -5,78 +5,91 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
+    <!-- Profile Image -->
     <ImageView
-        android:id="@+id/entrant_home_view_image_view"
+        android:id="@+id/profileImage"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginTop="32dp"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
-        tools:srcCompat="@tools:sample/avatars" />
+        tools:srcCompat="@tools:sample/avatars[5]" />
 
+    <!-- Username Title Text -->
     <TextView
-        android:id="@+id/entrant_home_view_user_name"
+        android:id="@+id/username"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginTop="16dp"
-        android:text="Name"
+        android:text="Username Placeholder"
+        android:textSize="18sp"
+        android:textStyle="bold"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/entrant_home_view_image_view" />
+        app:layout_constraintTop_toBottomOf="@+id/profileImage" />
 
-    <ScrollView
-        android:id="@+id/entrant_home_view_upcoming_events"
-        android:layout_width="0dp"
-        android:layout_height="200dp"
+    <!-- Upcoming Events Title -->
+    <TextView
+        android:id="@+id/upcoming_events_title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
         android:layout_marginStart="16dp"
         android:layout_marginTop="24dp"
         android:layout_marginEnd="16dp"
+        android:text="Upcoming Events"
+        android:textColor="#333333"
+        android:textSize="20sp"
+        android:textStyle="bold"
         app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/entrant_home_view_user_name">
+        app:layout_constraintTop_toBottomOf="@id/username" />
```

Comment:
we won't be using username, please leave this id unchanged

Reply:
We will be using name not username due to project requirements

Reply:
I believe I did this